applications/hpf/mspi/README.rst

```diff
@@ -0,0 +1,381 @@
+.. _hpf_mspi_example:
+
+High-Performance Framework MSPI
+###############################
+
+.. caution::
+
+   The High-Performance Framework (HPF) support in the |NCS| is :ref:`experimental <software_maturity>` and is limited to the nRF54L15 device.
+
+This application demonstrates how to write a :ref:`High-Performance Framework (HPF) <hpf_index>` application and communicate with it.
+The application implements subset of Zephyr MSPI API.
+
+Application overview
+********************
+
+The MSPI HPF application is structured into the following main components:
+
+* The HPF application - Operates on the FLPR core and facilitates data transmission between the application core and the connected MSPI device.
+* The Hard Real Time (HRT) module - Runs on the FLPR core and facilitates data transmission between the FLPR core and the connected MSPI device.
+  The module emulates the MSPI hardware peripheral on the FLPR core, managing real-time data transmission and ensuring precise timing and synchronization of data transfer operations.
+* The MSPI driver - Operates on the application core and uses the Zephyr's scalable real-time operating system (RTOS) MSPI API for data and configuration transmission between the application and FLPR cores.
```

Comment:
ref to this file could be added here:
https://github.com/nrfconnect/sdk-nrf/blob/main/drivers/mspi/mspi_hpf.c